### DISCUSSION TITLE
Image Database storage

### DISCUSSION THREAD

**Discussion by NigelVanHattum** [mealie-recipes/mealie#3962] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

I am running mealie on my kubernetes cluster without a mounted disk to it. This causes a loss for all images whenever te application restarts. 

### Please provide a concise description of the feature that would resolve your issue.

I would like the option to store my images within my postgreSQL database. That way they can be stored outside my cluster. 

### Please consider and list out some caveats or tradeoffs made in your design decision

This would seriously increase the database usage of the application. So maybe we can add it as a optional parameter that enables this function. 

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by NigelVanHattum** [comment]
Since I'm also using only public images, an option could also be that the application stores the URL to the image. Picking it up from the internet instead of the server.

**Comment by ChipSkylark37** [comment]
As another variant of this, are we able to access the image storage of Mealie? 

I have images that I would like to tweak, and see that in the JSON the image identified, but unsure of how to access in a Docker Container. 

The reason that I want to access / modify the image is that the rotation of many images are 'wrong' and as much as I rotate and re-upload, a lot of the times they keep their sideways rotation, likely based on the orientation of how I took the picture initially.
